astropy/tests/helper.py

```diff
@@ -126,10 +126,17 @@ def assert_follows_unicode_guidelines(x, roundtrip=None):
             assert eval(repr_x, roundtrip) == x
 
 
-@pytest.fixture(params=[0, 1, -1])
+@pytest.fixture(params=[0, 1, 4, -1])
```

Comment:
Hmm does the -1 need to change to 5 here to match the doc?
```suggestion
@pytest.fixture(params=[0, 1, 4, 5])
```

Reply:
Ah, had missed that that was changed in the docstring as well in your suggestion! Actually I would tend to leave it at –1, just clarifying that that resolves to 5 for the versions stated. And then hoping whenever a new protocol might be introduced, that we notice in time to update the docs...

Reply:
I clarified the docstring.